.evergreen/ocsp/mock_ocsp_revoked.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
+# Use the CA as the OCSP responder (see https://community.letsencrypt.org/t/unable-to-verify-ocsp-response/7264)
+python3 ocsp_mock.py \
+  --ca_file ca_ocsp.pem \
+  --ocsp_responder_cert ca_ocsp.crt \
```

Comment:
Per the comment above, I'm using the CA as the OCSP responder, which according to the above link, is also what Let's Encrypt does. Doing so simplifies the certificate chain and seems to reduce the amount of setup needed for Java at the very least (obviating the need to explicitly trust a separate OCSP responder certificate).